src/query/expression/src/filter/select_value/mod.rs

```diff
@@ -40,11 +40,101 @@ impl<'a> Selector<'a> {
         mutable_false_idx: &mut usize,
         select_strategy: SelectStrategy,
         count: usize,
+    ) -> Result<usize> {
+        match op {
+            SelectOp::Equal => self.select_type_values::<T>(
```

Comment:
You can use macro to refactor this match, see `with_mappedhash_method`

Reply:
Great, thanks!